src/2_RTB/VerbaleEsterno_101123/content.typ

```diff
@@ -0,0 +1,67 @@
+
+- *Creazione di uno Sprint Backlog:* Verrà compilato uno Sprint Backlog che conterrà le User Stories rilevanti per la fase iniziale dello sviluppo del _Proof of Concept_ (POC). Questo Sprint Backlog servirà come guida per le attività e le priorità da osservare durante il primo Sprint.
+
+=== Chiarimenti tecnici
+La Proponente ha consigliato di sviluppare un singolo script in Python per simulare l'attività di uno o più dispositivi operanti in modalità asincrona. In questa fase iniziale, l'attenzione principale sarà sulla struttura e il funzionamento del meccanismo di simulazione, senza necessariamente prestare particolare attenzione al realismo dei dati in uscita. I dispositivi possono essere definiti all'interno di un file CSV seguendo un formato predefinito che include informazioni anagrafiche come:
+
+- *Nome o ID:* Un identificatore univoco per il dispositivo;
+- *Tipologia:* La categoria o la tipologia del dispositivo;
+- *Codice di fabbrica:* Un codice che identifica il produttore o la provenienza del dispositivo;
+- *Coordinate geospaziali:* Le coordinate geografiche che rappresentano la posizione del dispositivo.
+
+Questi dati anagrafici forniranno una base per la simulazione, consentendo al team di concentrarsi sulla corretta implementazione delle funzionalità asincrone e di creare un primo ambiente di test per il flusso di dati. Successivamente, sarà possibile affinare ulteriormente la simulazione per includere dati più realistici.
+
+=== Analisi dei requisiti
+La Proponente suggerisce di avviare una prima fase di analisi dei requisiti concentrandosi sulle esigenze degli utenti. In questo contesto, è essenziale delineare come gli utenti intendano osservare i dati provenienti dai sensori in modalità _near real-time_, analizzarli attraverso operazioni che riflettono query SQL per manipolare il database, e monitorare lo stato attuale dei sensori, tra altre funzionalità.
+
+Successivamente, l'analisi potrebbe estendersi per comprendere requisiti relativi alle prestazioni del sistema, alla qualità dei dati, alle funzioni di auto-affidabilità, alla scalabilità orizzontale e al parallelismo, alla sicurezza, e altri aspetti critici. Si consiglia, quindi, di effettuare una distinzione chiara tra i requisiti utente e quelli applicativi, prestando anche attenzione alla categorizzazione tra requisiti obbligatori e opzionali. Questo approccio garantirà una comprensione approfondita delle aspettative degli utenti e consentirà di definire una _roadmap_ chiara per lo sviluppo del sistema, con priorità chiaramente definite.
+
+=== Contatti con la Proponente
+Oltre agli incontri pianificati ogni due settimane per stabilire gli obiettivi dello Sprint successivo, la Proponente offre di organizzare anche incontri di SAL intermedi. La frequenza e la distribuzione di tali incontri saranno adattate alle esigenze del team, consentendo una maggiore flessibilità e la possibilità di affrontare tempestivamente eventuali questioni o necessità emergenti.
```

Comment:
forse non è proprio concorde con il paragrafo 2.1.1. dove vengono menzionati i SAL ma in maniera diversa.